docs/Actions.md

```diff
@@ -329,13 +329,13 @@ This hook calls `dataProvider.getOne()` when the component mounts.
 
 ```jsx
 // syntax
-const { data, isFetching, isLoading, error, refetch } = useGetOne(resource, id, options);
+const { data, isFetching, isLoading, error, refetch } = useGetOne(resource, { id }, options);
 
 // example
 import { useGetOne } from 'react-admin';
```

Comment:
```suggestion
import { useGetOne, Loading } from 'react-admin';
```

Reply:
No, it's just an example - we don't have a Loading component for that.